dummies/test_demitto-tenuis.py

```diff
@@ -0,0 +1,38 @@
+import json
+import boto3
+import os
+import stripe
+
+# API Keys - For testing security tools only - These are fake keys
+FIREBASE_API_KEY = "AIzanOo6BIRbTk7zpgsXnGYeg7vSFKoXBXAxLeo"
+GITHUB_API_KEY = "AKIAGL1KFKAHT6614OFJ"
+
+class pixelClient:
```

Comment:
This should be upper case.
```suggestion
class PixelClient:
```